.devcontainer/devcontainer.json

```diff
@@ -1,4 +1,5 @@
 {
+	"remoteUser": "node",
```

Comment:
For clarity, I changed the Dev Container to use the `node` user instead of `root` as it messes with the file permission on Linux host.